README.md

```diff
@@ -58,3 +59,34 @@ This network use an interface on the host `vboxnet<N>` which need to be delete m
 # https://www.virtualbox.org/manual/ch06.html#network_hostonly
 VBoxManage hostonlyif remove vboxnet0
 ```
+
+## Containerized launch
+
+The Docker container has a Libvirt client installed which launches the VMs on your host. The only requirement is to have Docker and Libvirt.
+
+- Check the Libvirt GID for your user the command `id`.
+
+- Build the image:
+
+```sh
+docker build -t libvirt-client Docker
+```
+
+- Run and enter the container with a bind mount to the current host directory :
+
+```sh
+docker run -it --name libvirt --rm \
+-v $PWD:/home/tdp/tdp-dev \
+-v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
+-v vagrant-boxes:/home/tdp/.vagrant.d/boxes \
+--env CONTAINER_UID=$(id -u) --env CONTAINER_GID=<libvirt GID> \
+libvirt-client bash
```

Comment:
Use `tdp-vagrant` for names and get the GID of `libvirt` group via a shell command.
```suggestion
docker run -it --name tdp-vagrant --rm \
-v $PWD:/home/tdp/tdp-dev \
-v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
-v vagrant-boxes:/home/tdp/.vagrant.d/boxes \
--env CONTAINER_UID=$(id -u) --env CONTAINER_GID=$(getent group libvirt | cut -d: -f3) \
tdp-vagrant bash
```